dub.selections.json

```diff
@@ -4,9 +4,10 @@
 		"libevent": "2.0.1+2.0.16",
 		"libev": "5.0.0+4.04",
 		"openssl": "1.1.4+1.0.1g",
+		"libdparse": "0.2.1",
```

Comment:
Adding this effectively would make libdparse non-optional when buillding DDOX directly, but I guess we can count on libdparse enough to make it non-optional (I'll have a second look at #104 and merge it next, including the actual non-optional change there).